src/proto/grpc/testing/control.proto

```diff
@@ -117,6 +117,9 @@ message ClientConfig {
   // If 0, disabled. Else, specifies the period between gathering latency
   // medians in milliseconds.
   int32 median_latency_collection_interval_millis = 20;
+
+  // Number of client processes. 0 indicates no restriction.
```

Comment:
"no restriction"? Does this mean that we'll spin up a number of processes equal to the number of hardware cores?

Reply:
No restriction means this scenario config doesn't have explicit constrain on number of processes. It can be one or many. I designed it this way instead of using a `bool`, so we have the ability to control the specific number of processes in future.